MapboxCoreNavigationTests/CLLocationTests.swift

```diff
@@ -57,27 +89,12 @@ class CLLocationTests: XCTestCase {
         XCTAssertEqual(fixLocation.accuracyHorizontal, nil)
         XCTAssertEqual(fixLocation.speed, nil)
         XCTAssertEqual(fixLocation.time, now)
-    }
-    
-    func testMBFixLocationToCLLocation() {
-        let coordinate = CLLocationCoordinate2D(latitude: 1, longitude: 2)
-        let now = Date()
-        let fixLocation = FixLocation(coordinate: coordinate,
-                                        time: now,
-                                        speed: nil,
-                                        bearing: nil,
-                                        altitude: -1,
-                                        accuracyHorizontal: nil, provider: "default")
-        
-        let location = CLLocation(fixLocation)
+        XCTAssertEqual(fixLocation.verticalAccuracy, 1)
         
-        XCTAssertEqual(location.coordinate.latitude, 1)
-        XCTAssertEqual(location.coordinate.longitude, 2)
-        XCTAssertEqual(location.timestamp, now)
-        XCTAssertEqual(location.speed, -1)
-        XCTAssertEqual(location.course, -1)
-        XCTAssertEqual(location.altitude, -1)
-        XCTAssertEqual(location.horizontalAccuracy, -1)
+        if #available(iOS 13.4, *) {
+            XCTAssertEqual(fixLocation.bearingAccuracy, 2)
```

Comment:
nit:
Extract values we check into const within "given" section:
```suggestion
            XCTAssertEqual(fixLocation.bearingAccuracy, expectedBearingAccuracy)
```
?

Reply:
Good note, updated test.